plugins/eos/gs-plugin-eos.c

```diff
@@ -1341,6 +1344,20 @@ gs_plugin_eos_blacklist_app_for_remote_if_needed (GsPlugin *plugin,
 	return do_blacklist;
 }
 
+static void
+gs_plugin_eos_remove_blacklist_from_usb_if_needed (GsPlugin *plugin,
+						  GsApp *app)
+{
+	if (!gs_app_has_category (app, "Blacklisted"))
+		return;
+	else if (gs_app_has_category (app, "Blacklisted") &&
```

Comment:
You don't need to check for this because you already have checked it before. It should be something like:
```
if (!gs_app_has_category (app, "Blacklisted") || !gs_app_has_category (app, "USB"))